salt/modules/timezone.py

```diff
@@ -18,14 +18,22 @@
 
 log = logging.getLogger(__name__)
 
+__virtualname__ = 'timezone'
+
 
 def __virtual__():
     '''
     Only work on POSIX-like systems
     '''
     if salt.utils.is_windows():
-        return (False, 'The timezone execution module failed to load: not available on Windows systems.')
-    return True
+        return (False, 'The timezone execution module failed to load: '
+                       'not available on Windows systems.')
+
+    if salt.utils.is_darwin():
+        return (False, 'The timezone execution module failed to load: '
+                       'not available on MacOS systems.')
```

Comment:
I understand why it isn't loaded. My point is that if we're going to write error messages, let's make them as helpful as they can be instead of just copying whatever else is nearby. :]